SQLHelper/pom.xml

```
@@ -32,4 +32,25 @@
     <maven.compiler.source>15</maven.compiler.source>
     <maven.compiler.target>15</maven.compiler.target>
   </properties>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-shade-plugin</artifactId>
+        <version>3.2.4</version>
+        <configuration>
+          <!-- put your configurations here -->
+        </configuration>
```

```suggestion
        <configuration>
              <relocations>
                <relocation>
                  <pattern>net.sf.jsqlparser</pattern>
                  <shadedPattern>org.chaosmesh.shaded.net.sf.jsqlparser</shadedPattern>
                </relocation>
              </relocations>
        </configuration>
```

We should use relocation to avoid class conflicts

---

done